### DISCUSSION TITLE
LICENSE file not detected on my repository

### DISCUSSION THREAD

**Discussion by hhromic** [community/community#59985] [Repositories] [answered]
### Select Topic Area

Bug

### Body

Hello,
The LICENSE file in one of my repositories is not being detected: <https://github.com/hhromic/promrelay-exporter>

As you can see in the following screenshot (in case it changes in the future), the License field is not present:

<img src="https://github.com/community/community/assets/5802993/0b63f613-f30d-4bcd-8c2c-80aeec383ff1" width="40%">

All of my other repositories work as expected.

Something that _might_ be related is that I have renamed this repository two times, one some days ago and another time a couple of years ago. I cannot remember if the license was being shown before though.

Related discussions (but not exactly the same):
* https://github.com/orgs/community/discussions/53739
* https://github.com/orgs/community/discussions/22058

I don't believe I'm doing something wrong, or is this a bug in GitHub?

**Comment by 3urobeat** [comment, marked as answer]
Good question, GitHub does recognize your repo contains a LICENSE in the [community standards page](https://github.com/hhromic/promrelay-exporter/community).  
Compared to the licenses in your other repos this also has some kind of syntax highlighting for me, very weird.

Stupid question, but: Have you tried recreating it by deleting and readding it through the repository community standards page? Maybe that causes GitHub to re-detect it, idk.

**Comment by hhromic** [comment]
> Good question, GitHub does recognize your repo contains a LICENSE in the [community standards page](https://github.com/hhromic/promrelay-exporter/community).

Ah yes, I forgot to mention that indeed, that GitHub recognises "a license" in there.
As a further insight, the LICENSE file in this [other repo of mine](https://github.com/hhromic/traefik-oidcintro-fwdauth) is exactly the same file and it is recognised by GitHub.

> Stupid question, but: Have you tried recreating it by deleting and readding it through the repository community standards page? Maybe that causes GitHub to re-detect it, idk.

No, I did not try that, but I just did now and it worked!  🎉💯
I just removed the existing LICENSE file from the repository with a commit, and re-added it using GitHub in another commit.

The newly added file has the exact same hash than the previously existing one, so I guess it was a GitHub glitch :)
Thanks anyway for the help/tip! 👍
